src/common/monitors.cpp

```diff
@@ -1,84 +1,64 @@
 #include "pch.h"
 
-#include "common.h"
 #include "monitors.h"
 
+#include "common.h"
+
 bool operator==(const ScreenSize& lhs, const ScreenSize& rhs)
 {
     auto lhs_tuple = std::make_tuple(lhs.rect.left, lhs.rect.right, lhs.rect.top, lhs.rect.bottom);
     auto rhs_tuple = std::make_tuple(rhs.rect.left, rhs.rect.right, rhs.rect.top, rhs.rect.bottom);
     return lhs_tuple == rhs_tuple;
 }
 
-static BOOL CALLBACK get_displays_enum_cb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
+static BOOL CALLBACK getDisplaysEnumCb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
```

Comment:
nit: Let's not keep adding/mixing coding styles :) `common` is already mixed with UpperCamelCase and `underscore_case`. So either start method name with UpperCase or convert it to `underscore_case` . I personally prefer `UpperCamelCase`

Same for the following methods

Reply:
Renamed